src/main/kotlin/io/sip3/salto/ce/sip/SipMessageHandler.kt

```diff
@@ -194,19 +194,8 @@ open class SipMessageHandler : AbstractVerticle() {
     }
 
     open fun routeSipMessage(prefix: String, packet: Packet, message: SIPMessage) {
-        val route = when (prefix) {
-            RoutesCE.sip + "_call" -> {
-                val index = message.callId().hashCode()
-                prefix + "_${abs(index % instances)}"
-            }
-            RoutesCE.sip + "_register" -> {
```

Comment:
We lost this very important moment of routing of REGISTERS, which has to be moved to transaction handler... Otherwise we will send transactions for one and the same session to different SipRegisterHandler intances